includes/admin/settings/display-settings.php

```diff
@@ -27,10 +27,13 @@ function edd_options_page_primary_nav( $active_tab = '' ) {
 		<?php
 
 		foreach ( $tabs as $tab_id => $tab_name ) {
-			$tab_url = add_query_arg( array(
-				'settings-updated' => false,
-				'tab'              => $tab_id,
-			) );
+			$tab_url = add_query_arg(
+				array(
+					'settings-updated' => false,
+					'tab'              => $tab_id,
```

Comment:
```suggestion
					'post_type'        => 'download',
					'page'             => 'edd-settings',
					'tab'              => $tab_id,
```

I hope the spacing is right lol. It isn't lining up in the `Write` view but is in the `Preview` :P

Reply:
@ashleyfae actually I think we can remove the `post_type` altogether from both of these as it's added in the `edd_get_admin_base_url` function?